### PR TITLE
[14.0][IMP][partner_identification] Allow attachments and chatter for res_partner_id_number model

### PR DESCRIPTION
**Pull request by cvinh**
With this commit, one can attach documents in chatter and plan activities in res_partner_id_number model

### CODE REVIEW

File: partner_identification/__manifest__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 {
     "name": "Partner Identification Numbers",
     "category": "Customer Relationship Management",
-    "version": "14.0.1.2.0",
+    "version": "14.0.1.3.0",
```
Comment (ivantodorovich) [inline partner_identification/__manifest__.py]: This is not necessary, the bot will bump the version upon merge

**Review by ivantodorovich — APPROVED** [review]

**Comment by cvinh** [comment]
Hello, is there a psc member who can apply 'no stale' please ?

**Comment by pedrobaeza** [comment]
Label removed. Get other reviewer reviewing other PRs and asking in exchange that the review yours.

**Comment by cvinh** [comment]
Hello, is there a psc member who can apply 'no stale' please ?

**Review by rousseldenis — APPROVED** [review]
Seems great

**Comment by rousseldenis** [comment]
@cvinh Could you check tests ?

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM

**Comment by rousseldenis** [comment]
@cvinh

**Comment by cvinh** [comment]
> @cvinh Could you check tests ?

It seems that Tests fail because of animal module
@rousseldenis do you confirm ?

**Comment by rousseldenis** [comment]
@cvinh Could you rebase as I think animal module has been fixed?

**Comment by cvinh** [comment]
> @cvinh Could you rebase as I think animal module has been fixed?
Thanks for pointing that
Yes it's fixed
@rousseldenis you might remove the 'need fixing' label or is there something else to do on my side ?

**Pull request merged**